backends/arm/test/ops/test_mean_dim.py

```diff
@@ -106,7 +106,12 @@ def _test_meandim_tosa_u55_BI_pipeline(
             .check(["torch.ops.quantized_decomposed"])
             .to_edge()
             .partition()
```

Comment:
can we check for the avg_pool2d to make sure the pass worked correctly (at least where it should)?

Reply:
Well, not here. The pass is internal to the Arm Backend, so the graph will be serialised to TOSA fb and compiled and included in the delegate call. So will add a separate test for the pass itself in addition to this test.

Reply:
Not sure I follow. What I mean is before partition we should have `avg_pool2d` in the graph so that we can replace it.